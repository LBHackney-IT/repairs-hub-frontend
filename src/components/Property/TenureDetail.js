import PropTypes from 'prop-types'

const TenureDetail = ({ text, detail }) => (
  <>
    {detail && (
      <li className="bg-orange">
        {text}: {detail}
      </li>
    )}
  </>
)

TenureDetail.propTypes = {
  detail: PropTypes.string,
  text: PropTypes.string,
}

export default TenureDetail
