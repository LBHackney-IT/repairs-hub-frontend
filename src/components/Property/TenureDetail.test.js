import { render } from '@testing-library/react'
import TenureDetail from './TenureDetail'

describe('TenureDetail component with TMO name', () => {
  const props = {
    detail: 'Test address TMO',
    text: 'TMO',
  }

  it('should render properly', async () => {
    const { asFragment } = render(
      <TenureDetail
        text={props.text}
        detail={props.detail}
      />
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
