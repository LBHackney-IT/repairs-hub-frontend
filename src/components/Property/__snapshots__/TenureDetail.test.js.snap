// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TenureDetail component with TMO name should render properly 1`] = `
<DocumentFragment>
  <li
    class="bg-orange"
  >
    TMO: Test address TMO
  </li>
</DocumentFragment>
`;
